### PR TITLE
Add a font package to enable r-ggplot2 in containers.

### PR DESCRIPTION
**Pull request by rhpvorderman**
This is necessary because in most containers there are no font packages installed. Any plots generated by r-ggplot2 on these systems will just contain square symbols instead of actual text. 

This is a major issue for biocontainers.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rhpvorderman** [comment]
@conda-forge-admin, please rerender

**Comment by cbrueffer** [comment]
Is this only an issue for ggplot2? What about e.g. r-base?

**Comment by rhpvorderman** [comment]
I am not at all an R user. I only run tools created by R. Currently all the tools that I have run and run into this issue had ggplot2 as a dependency. 
This will be an issue for any package that renders fonts. If R-base is also capable of rendering plots with fonts, then I can move that PR there if you feel it is more appropriate.

**Comment by cbrueffer** [comment]
Sounds like this would be best added to the `r-base` recipe then, that way it'll be automatically available for all R packages. What do you think @conda-forge/r ?

**Comment by dbast** [comment]
There is lots of context to this... see https://github.com/conda-forge/r-base-feedstock/issues/91

tldr; r-base depends on fontconfig, fontconfig contains fonts-conda-ecosystem as run_exports, fonts-conda-ecosystem depends on fonts-conda-forge  and that depends on font-ttf-dejavu-sans-mono, font-ttf-inconsolata, font-ttf-source-code-pro, font-ttf-ubuntu 

that means this `conda create --name test r-ggplot2` install the fonts required by fonts-conda-forge.

If fonts-conda-forge is not complete then please open a PR against https://github.com/conda-forge/fonts-conda-forge-feedstock instead of adding fonts to individual r packages.

**Comment by rhpvorderman** [comment]
@dbast, thank you for the information. I have run into conda environments in containers were the fonts were not installed, and I incorrectly assumed that was because there were no fonts as dependencies. It seems the problem lies with the container creation then. 

I am sorry for the inconvenience.
